webui/react/src/pages/ExperimentDetails/ExperimentVisualization/ExperimentVisualizationFilters.tsx

```diff
@@ -185,7 +185,7 @@ const ExperimentVisualizationFilters: React.FC<Props> = ({
             label="Batches Processed"
             searchable={false}
             value={filters.batch}
-            width={70}
+            width={100}
```

Comment:
I think removing the `width` prop entirely makes it a dynamic width based on content

Reply:
removing width makes select component dynamic, but that causes other issue

<img width="233" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/a6a643d0-769d-4e0c-b620-84d94009bc60">

Reply:
OK, I think these changes are fine to address the reported issue in the short-term. But I think we've seen this come up enough that it warrants a long-term solution. IMO it doesn't really make sense to manually set the width for each of these, especially for string values where we can't really predict the length.

Since you've dealt with this in a few places now, do you mind putting together a ticket that captures requirements for dynamic width behavior in the Select component? We might need to reimplement parts of it without antd, not sure. But I think the selected option and the options list should both have dynamic widths by default (and a fixed width for selected option should allow the options list to still be dynamic width).

Reply:
Made (a ticket](https://hpe-aiatscale.atlassian.net/browse/WEB-1565)